sys/docker/alpine-phpfpm/Dockerfile

```diff
@@ -53,6 +53,8 @@ ENV LD_PRELOAD /usr/lib/preloadable_libiconv.so php
 
 COPY --from=composer:2 /usr/bin/composer /usr/local/bin/composer
 
+RUN composer self-update
```

Comment:
sure?

Reply:
Installing dependencies from zero returns a errors, we need to keep composer updated.

Reply:
also after update composer.lock anche symfony.lock?

Reply:
and so, if we commit composer.lock and symfony.lock updated, we can remove this line?

Reply:
I don't think so, we should install composer 2.1 into our docker image if we would like to remove this line.

Reply:
`COPY --from=composer:2.1 /usr/bin/composer /usr/local/bin/composer`
don't work?

Reply:
I can try, it could be better.

Reply:
Maybe we could force the minor version of composer? Last one is 2.3

Reply:
Without `composer self-update` I had problems with dependencies. So we can try to remove the line and commit `composer.lock` and `symfony.lock`.

Reply:
I would add those files to git :+1:

Reply:
Those 2 files are already committed

Reply:
I removed the `composer self-update` at the moment. I verified and now it seems to work fine.